Add instructions to close the app (#190)
Princeton-CDH/remarx: README.md

uv run remarx-app
```

### Closing quote finder app

> The app will not close automatically when its browser window is closed.
> To close the app:
> To close the app:

Reviewer: ```suggestion
The app will not close automatically when you close the browser window or tab.
To close the app:
```
